src/types/role.ts

```diff
@@ -7,4 +7,14 @@ export interface RolePayload {
   permissions: string
   managed: boolean
   mentionable: boolean
+  tags: RoleTagsPayload | null
```

Comment:
[From the discord docs](https://discord.com/developers/docs/topics/permissions#role-object) it should be undefined instead of null